backend/open_webui/retrieval/vector/dbs/oracle23ai.py

```
@@ -0,0 +1,943 @@
+
+        Returns:
+            Optional[GetResult]: Result containing ids, documents, and metadata
+
+        Example:
+            >>> client = Oracle23aiClient()
+            >>> results = client.get("my_collection", limit=50)
+            >>> if results:
+            ...     print(f"Retrieved {len(results.ids[0])} documents from collection")
+        """
+        log.info(
+            f"Getting items from collection '{collection_name}' with limit {limit}."
```

Ruff (0.12.2)

Undefined name `limit`

(F821)